### ISSUE TITLE
Not launching ?

### ISSUE THREAD

**Issue by tazeps** [mmpx12/twitter-media-downloader#43]
I downloaded the latest release (windows AMD v1.12) and when I launch it nothing happens.

I tried opening up command prompt and using the the command twmd as per the example bu it's not working. (it says command is not recognised).

I also tried changing the directory to the location where the executable is in but i get the same result.

Any idea why this might be?

I tried the GUI too but when I input my parameters and click "download" it processes for a while then stops without any apparent progress or any results.

**Comment by tazeps** [comment]
aaah ok I understand what you mean. the exe file is actually named "twitter-media-downloader.exe" but I understand what you mean and I tried it. 

It successfully launched when I run it from command prompt. however I get mixed results when I try to download media. It either doesn't download anything at all, or downloads a seemingly random number of media. See attached Screenshot.

Also, I tried to log in (-L), I enter the username, but when I tryto enter the password, it doesn't detect any input from keyboard. I also can't copy/paste my password

![image](https://github.com/mmpx12/twitter-media-downloader/assets/33405597/43b25f37-b49d-45b6-b084-ba442c0b9d14)
